### PR TITLE
Full hsbet module removal

### PR DESCRIPTION
**Pull request by ALazyMeme**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [x] Documentation in docs/ or install-docs/ was updated, if applicable
- [ ] I have tested all changes

### CODE REVIEW

**Review by pajlada — APPROVED** [review]
Looks good to me, feel free to merge it in when you've given it a little test!

File: pajbot/migration_revisions/db/0016_delete_hsbet_module.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def up(cursor, bot):
+    cursor.execute("DROP TABLE hsbet_game, hsbet_bet")
```
Comment (RAnders00) [inline pajbot/migration_revisions/db/0016_delete_hsbet_module.py:2]: Delete the enum type as well: https://github.com/pajbot/pajbot/blob/7aea8a15fb13cc83295c46e1751965bdf496f6bf/pajbot/migration_revisions/db/0001_initial.py#L115

**Review by RAnders00 — CHANGES_REQUESTED** [review]

**Review by RAnders00 — APPROVED** [review]

**Comment by RAnders00** [comment]
Same as pajlada, i think you can merge this once you‘ve tested it

**Comment by ALazyMeme** [comment]
Little test has been completed, this works as expected.

**Pull request merged**